server/src/unifyfs_server.c

```diff
@@ -640,6 +651,14 @@ app_config* new_application(int app_id)
     for (int i = 0; i < MAX_NUM_APPS; i++) {
```

Comment:
Overall, this looks good to me. Just one question in this loop. If we cannot find any empty slot for the new app client request, is there a reason that we need to keep the old app data structures, instead of victimizing the oldest one and allowing the new app to connect; e.g, is it necessary to locate data (in logio)?

Reply:
MAX_NUM_APPS is still hard-coded, and represents the maximum number of application mountpoints. If you run k apps all with the same mountpoint (e.g. '/unifyfs'), it only counts as one app_config. The new configuration setting controls the maximum number of client processes that can mount at the same mountpoint, regardless of how many client applications you run. The new default is 256, so to exhaust that you could do 4 client applications each with 64 ppn, or 8 with 32ppn, or 16 with 16ppn.

Reply:
Now I understand. Thanks for explanation!